File: people/src/test/java/com/young/ssh/TestUserTable.java
package com.young.ssh;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.cfg.Configuration;
import org.hibernate.service.ServiceRegistry;
import org.hibernate.service.ServiceRegistryBuilder;
import org.hibernate.tool.hbm2ddl.SchemaExport;
import org.junit.Test;


/**
 * Created by young on 2017/11/2.
 */
public class TestUserTable {

    @Test
    public void createTable() {
        Configuration configuration = new Configuration().configure();
        ServiceRegistry serviceRegistry = new ServiceRegistryBuilder()
                .applySettings(configuration.getProperties()).buildServiceRegistry();
        SessionFactory sessionFactory = configuration.buildSessionFactory(serviceRegistry);

        Session session = sessionFactory.getCurrentSession();

        SchemaExport export = new SchemaExport(configuration);

        export.create(true, true);
    }
}

File: people/src/test/java/com/young/ssh/TestUserTable.kt
package com.young.ssh

import org.hibernate.cfg.Configuration
import org.hibernate.service.ServiceRegistryBuilder
import org.hibernate.tool.hbm2ddl.SchemaExport
import org.junit.After
import org.junit.Before
import org.junit.Test


/**
 * Created by young on 2017/11/2.
 */
class TestUserTable {

    @Before
    fun beforeInfo() {
        println("准备初始化数据库")
    }

    @Test
    fun createTable() {
        val configuration = Configuration().configure()
        val export = SchemaExport(configuration)
        export.create(true, true)
    }

    @After
    fun afterInfo() {
        println("数据库初始化完成")
    }
}